src/pages/ExamsPage/index.test.jsx

```diff
@@ -20,6 +21,7 @@ jest.mock('./hooks', () => ({
 describe('ExamsPage', () => {
   beforeAll(() => {
     hooks.useExamAttemptsData.mockReturnValue(testUtils.defaultAttemptsData);
+    hooks.useAllowancesData.mockReturnValue({ allowancesList: [] });
```

Comment:
Question: I took a look at the epic and it looks like actually having allowances is not in the scope of this work, right? That is, for now, we only need to test this case and not the case where there are allowances? Thanks for clarifying.